assessments/0008-backstage/user-roles.md

```diff
@@ -0,0 +1,31 @@
+# Backstage User Roles for Doc
+
+This document provides recommendations for user roles, or personas, for Backstage. The goal here is to identify a working set of distinct user roles around which to organize the Backstage documentation. 
+
+The documentation should ultimately be task-oriented, with tasks organized around users and their objectives. A process for creating this type of documentation set is under development in the CDCF Tech Doc GitHub project. 
+
+## Summary of Proposed User Roles
+
+The following table summarizes Backstage user roles proposed by the Backstage OSS community\*. The roles assume the following:
+
+- An organization has software objectives that cannot be met without a centralized solution, and has resolved to commit resources to solving those problems.
+- The organization adopts Backstage, setting up a single, central Backstage server instance ("app").
+- The organization contains many development groups. These groups can be organized in any manner, but this document assumes a flat organization of small development groups.
+- "Developmenet group" refers to any group that produces a software product manageable in Backstage, including but not limited to internal and external toolkits and APIs; components; databases; and web-based and standalone applications.
+- The organization has ties to the Backstage open source software project, specifically:
+    - One or more engineers who contribute to the project.
+    - Developer users who ask questions and participate in discussions, newsgroups, and other community forums.
+
+\*Ultimately. This document is a work in progress to be revised by consensus.
+
+| User Role | Org Level | Description |
+| --------- | --- |  ----------- |
+| Champion  | Organization |  A combination evangelist/implementer/coordinator dedicated to driving adoption of Backstage at an organization. The champion is vital to removing barriers to developer adoption by, among other things, making the developer experience a delight and by demonstrating Backstage's value. |
```

Comment:
Is the Champion the same as a "decision maker?" Are these the folks that choose which software to use, or do they advise the decision makers? Do they need different information from Administrators when deciding whether or not to use Backstage?

Reply:
If you mean "decision maker" in the sense of "keeper of purse strings", then not necessarily -- the champion might be a technical leader without budget authority. But I think the champion should have the authority to make technical decisions about the installation. It's a hard job; why make it harder by tying their hands?

Reply:
I think of the decision maker as someone with the authority to say "we're doing this." I think I may be straying into conversations about the adoption process but if we're enumerating website users, it may be worth keeping them in mind even if it's just while writing up things like the getting started guide.